code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -14,6 +14,8 @@
 	required_candidates = 1
 	weight = 5
 	cost = 10
+	var/max_traitors = 3
```

Comment:
A non-concern but max_traitors is non-indicative of what this does, maybe change it to threshold_traitors? Ditto for vampires